lib/registration.pm

```diff
@@ -93,6 +93,8 @@ sub accept_addons_license {
     push @addons_with_license, @SLE15_ADDONS_WITHOUT_LICENSE unless is_sle('15+');
     # HA and WE have licenses when calling yast2 scc
     push @addons_with_license, @SLE15_ADDONS_WITH_LICENSE_NOINSTALL if (is_sle('15+') and get_var('IN_PATCH_SLE'));
+    # Live patching do not have license when calling yast2 scc
+    @addons_with_license = grep { $_ ne 'live' } @addons_with_license if (is_sle('15+') && get_var('IN_PATCH_SLE'));
```

Comment:
this line is a little duplicated, how about
```
our @SLE15_ADDONS_WITHOUT_LICENSE = qw(ha sdk wsm we hpcm live);
...
my @addons_with_license = qw(geo rt idu ids lgm ses);
```
then
`push @addons_with_license, @SLE15_ADDONS_WITHOUT_LICENSE unless is_sle('15+');`
will take effect.

Reply:
Updated following your comments. Thanks. 
Verified log: http://openqa-apac1.suse.de/tests/2538